TestAlterTiering::test[many_tables] fails in setup_class

Падает при запуске на olap-testing 
[fail] test_alter_tiering.py::TestAlterTiering::test[many_tables] [default-linux-x86_64-profile] (0.09s)
setup failed:
ydb/tests/olap/scenario/conftest.py:70: in setup_class
    ScenarioTestHelper(None).remove_path(cls.get_suite_name())
ydb/tests/olap/scenario/helpers/scenario_tests_helper.py:707: in remove_path
    self._run_with_expected_status(
ydb/tests/olap/scenario/helpers/scenario_tests_helper.py:353: in _run_with_expected_status
    pytest.fail(f'Unexpected status: must be in {repr(expected_status)}, but get {repr(error or status)}')
E   Failed: Unexpected status: must be in {<StatusCode.SUCCESS: 400000>}, but get SchemeError('message: "Check failed: path: \\\'/olap-testing/kikimr/testing/testdb2/scenario/TestAlterTiering/many_tables\\\', error: path has children, request doesn\\\'t accept it, children: 2" severity: 1 (server_code: 400070)')

--- a/ydb/tests/olap/scenario/helpers/scenario_tests_helper.py
+++ b/ydb/tests/olap/scenario/helpers/scenario_tests_helper.py
@@ -706,6 +706,8 @@ class ScenarioTestHelper:
                 self.execute_scheme_query(dh.DropTable(os.path.join(folder, e.name)))
             elif e.is_column_store():
                 self.execute_scheme_query(dh.DropTableStore(os.path.join(folder, e.name)))
+            elif e.is_external_data_source():
+                self.execute_scheme_query(dh.DropExternalDataSource(os.path.join(folder, e.name)))
             elif e.is_directory():
                 self._run_with_expected_status(
                     lambda: YdbCluster.get_ydb_driver().scheme_client.remove_directory(os.path.join(root_path, e.name)),

